packages/devp2p/src/protocol/eth.ts

```diff
@@ -299,8 +297,6 @@ export class ETH extends Protocol {
         if (this._version >= ETH.eth62.version) break
         throw new Error(`Code ${code} not allowed with version ${this._version}`)
 
-      case ETH.MESSAGE_CODES.GET_NODE_DATA:
-      case ETH.MESSAGE_CODES.NODE_DATA:
```

Comment:
Hiya, we will need access to these message codes if we want to run anything `< eth/67`

Reply:
shouldn't this also be if `>= eth/63`?

Reply:
Yes, so any anything between 63 and 66 should be able to access it.

Reply:
fixed it & added a test to show methods not reachable on `eth/67`